src/components/FAQ/faq.md.js

```diff
@@ -12,6 +12,10 @@ export const faq = `## Frequent Questions
   * The CAMPD app user interface limits downloads from the streaming services to 1,000,000 records but there is no download records limit on the streaming services APIs.
 
 ---
+### 2) Where can I find frequent questions about the CAM API data?
+
+  * Frequent questions on the data in the CAM API can by found in the [CAMPD FAQs](https://campd.epa.gov/help-support/faqs) page.
```

Comment:
@sgillEPA - How about something like this 
"2) Where can I find frequent questions regarding the power plant emissions, compliance, and allowance data available in CAMPD?
Frequent questions about data availability can by found in the [CAMPD FAQs](https://campd.epa.gov/help-support/faqs) page."